src/System.Management.Automation/engine/NativeCommandProcessor.cs

```diff
@@ -432,19 +446,10 @@ private void InitNativeProcess()
 
                     // Also, store the Raw UI coordinates so that we can scrape the screen after
                     // if we are transcribing.
-                    try
-                    {
-                        if (this.Command.Context.EngineHostInterface.UI.IsTranscribing)
-                        {
-                            _scrapeHostOutput = true;
-                            _startPosition = this.Command.Context.EngineHostInterface.UI.RawUI.CursorPosition;
-                            _startPosition.X = 0;
-                        }
-                    }
-                    catch (Host.HostException)
```

Comment:
Looks like an HostException also indicates `The host doesn't support scraping via its RawUI interface` ...